app/codelab/scaffold-app.md

```diff
@@ -48,42 +44,24 @@ If you have a few generators installed, you'll be able to interactively choose f
 
 Some generators will also provide optional settings to customize your app with common developer libraries to speed up the initial setup of your development environment.
 
-The AngularJS generator provides options to use [Sass](http://sass-lang.com/) (with [Compass](http://compass-style.org)) and include [Twitter Bootstrap](http://getbootstrap.com/). For this codelab, we **won't** use Sass but **will** use Bootstrap. Enter `n` then `y` respectively to these options.
-
-![](/assets/img/codelab/image_8.png)
-
-Next, you are prompted to select what Angular modules you would like to include as well:
-
-![](/assets/img/codelab/image_9.png)
-
-Angular modules are self-contained JavaScript files with helpful functionality. For example, the ngResource module (*angular-resource.js*) provides interaction support with RESTful services.
+The FountainJS generator provides some choices to use your favorite:
+* framework ([React](https://facebook.github.io/react/), [Angular2](https://angular.io/) or [Angular1](https://angularjs.org/))
```

Comment:
The list is not rendering as a list
<img width="891" alt="capture d ecran 2016-06-13 a 09 53 38" src="https://cloud.githubusercontent.com/assets/10107323/16000435/bc976ee6-314c-11e6-961e-2a345ddcf3c5.png">